code/game/jobs/job/civilians.dm

```diff
@@ -179,10 +179,10 @@ Your role involves a lot of roleplaying, but you can perform the function of a r
 
 
 	generate_entry_message(mob/living/carbon/human/H)
-		. = {"As a representative of Weyland-Yutani Corporation, your job requires you to stay in character at all times.
+		. = {"As a representative of Weyland-Yutani Corporation you are expected to stay professional and loyal to the corporation at all times.
```

Comment:
Could you replace WY with NT?

Reply:
Yes, but on a separate PR. There are way too many istances in the game, plus a UI background, logos and icons. Doing this little change here would just confuse someone a little.

Reply:
Fair.